<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateAirplane</class>
 <widget class="QMainWindow" name="UpdateAirplane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_field_registration">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>&gt;AA-AAA</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_manufacturer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_manufacturer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manufacturer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_model">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_copilot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_model">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_capacity">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit the desired fields</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_pilot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_capacity">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>900</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_pilot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pilot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_registration">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_copilot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Co-Pilot</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_update_airplane">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
